angular/demo/src/app/samples/rating/form.route.ts

```diff
@@ -1,17 +1,20 @@
 import {AgnosUIAngularModule} from '@agnos-ui/angular';
-import {NgIf} from '@angular/common';
 import {Component} from '@angular/core';
 import {FormControl, ReactiveFormsModule, Validators} from '@angular/forms';
 
 @Component({
 	standalone: true,
-	imports: [AgnosUIAngularModule, ReactiveFormsModule, NgIf],
+	imports: [AgnosUIAngularModule, ReactiveFormsModule],
 	template: `
 		<label class="form-label" id="ratingLabel">Rating of your experience</label><br />
 		<div [auRating] [formControl]="ctrl" auAriaLabelledBy="ratingLabel"></div>
 		<div id="form-msg" class="form-text small">
-			<div *ngIf="ctrl.valid" class="text-success">Thanks!</div>
-			<div *ngIf="ctrl.invalid" class="text-danger">Please rate us</div>
+			@if (ctrl.valid) {
```

Comment:
maybe a @else only here ?

Reply:
@ExFlo for info this will change the behavior when the control is disabled, and in this sample we provide a button to disable it. It would become a different behavior compared to svelte or react, so I won't add the @else here